### ISSUE TITLE
Make it harder to accidentally overwrite the socket file

### ISSUE THREAD

**Issue by zormit** [ethersync/ethersync#104]
Currently we're swapping quickly between different projects and sometimes run daemons in parallel for them. The default socket `/tmp/ethersync` is removed at start of a daemon, so if you forget to provide the optional parameter, it might "steal" it from a running daemon, if you start the second one.

What should happen instead: If the socket file exists, refuse to start. But clean up your own socket file during shutdown.

**Comment by blinry** [comment]
No longer relevant thanks to #276.
